public/js/user/home.js

```diff
@@ -35,6 +37,7 @@ formContactUs.addEventListener('submit', (e) => {
       if (res.msg === 'done') {
         popup.classList.add('popup-show');
         popupMassage.innerText = 'We received your message, we will send you our response soon.';
+        innerTextRemove(Array.from(inputs).slice(0, 3));
```

Comment:
@aminalakhsham Instead of using a slice and trying to get the numbers right, you could use a more specific querySelectorAll() here, and then a different one further down on line 69 for the email element. For  example, here could be “document.querySelectorAll(‘.footer—contactUs-form .footer-inputs)” and line 69 could be “document.querySelectorAll(‘.footer—form .footer-inputs’)”. 
That would make this code a little easier to understand for other devs, and also would still continue to work if you added or removed form fields from those forms in your markup.